src/api/resolvers/types/mutations/inputs/GraphQLCreateIMSInput.ts

```diff
@@ -0,0 +1,40 @@
+import { GraphQLInputFieldConfig, GraphQLInputObjectType, GraphQLInputObjectTypeConfig, GraphQLString, GraphQLNonNull, GraphQLList, GraphQLID } from "graphql";
+import GraphQLJSON from "../../../scalars/GraphQLJSON";
+import GraphQLIMSType from "../../../enums/GraphQLIMSType";
+
+let createIMSInputConfig: GraphQLInputObjectTypeConfig = {
+    name: "CreateIMSInput",
+    description: "The inputs for the createIMS mutation",
+    fields: () => ({
+        clientMutationID: {
+            type: GraphQLString,
+            description: "An arbitraty string to return together with the mutation result"
+        },
+        imsType: {
+            type: GraphQLIMSType,
+            description: "The type/system this IMS is an instance of"
+        },
+        endpoint: {
+            type: GraphQLString,
+            description: `The endpoint where to reach the IMS instance.
+            
+In the most cases this will be a URL in the form of
+\`\`\`
+https://example.com/api/[API_KEY]

```

Comment:
sure?
we want to make the changes depending on the user, so there should probably be no api key there

Reply:
Maybe bad description. Changed it in 78154e3